ui/src/app/views/workspace/component.ts

```diff
@@ -32,7 +35,22 @@ export class WorkspaceComponent implements OnInit {
     this.cohortsService.getCohortsInWorkspace('123', '123').subscribe(
           cohortsReceived => {
             this.cohortList = cohortsReceived;
+            this.CohortEditService.list().then(
+              cohorts => {
+                for (const coho of cohorts) {
+                  this.cohortList.push(coho);
+                }
+              }
+            );
           });
     this.currentUrl = this.router.url;
   }
+
+  addCohort(): void {
+    this.router.navigate([this.currentUrl + '/cohort/' + this.CohortEditService.add()]);
```

Comment:
Beyond the demo, we won't have an ID at the time a user picks "new", right? Because ids are allocated at the time a name is assigned, which is after the interaction with the cohort builder?